backend/FwLite/MiniLcm/Validators/PartOfSpeechIdValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+
+namespace MiniLcm.Validators;
+
+public class PartOfSpeechIdValidator : AbstractValidator<Guid?>
```

Comment:
if we do end up keeping this then I might change the name to `IsCanonicalPartOfSpeechIdValidator` as it doesn't just validate any Id, it's validating that's it's canonical.

Reply:
Will do later.

Reply:
Done in commit 0a39681a. We might remove it later but the rename is simple so it's worth doing now even if we do remove it later.